packages/lit/lit@2.1.1.web-types.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "../../schema/web-types.json",
+  "framework": "lit",
+  "name": "lit",
+  "version": "2.1.1",
+  "js-types-syntax": "typescript",
+  "description-markup": "markdown",
+  "default-icon": "icons/lit.svg",
+  "framework-config": {
+    "enable-when": {
+      "node-packages": [
+        "lit"
+      ]
+    }
+  },
+  "contributions": {
+    "html": {
+      "attributes": [
+        {
+          "name": "Event listeners",
+          "description": "Event listeners expression",
+          "virtual": true,
+          "doc-url": "https://lit.dev/docs/templates/expressions/#event-listener-expressions",
+          "value": {
+            "kind": "expression",
+            "type": "(event: Event) => void",
+            "required": true
+          },
+          "pattern": {
+            "items": "/js/events",
+            "template": [
+              "@",
+              "$...",
+              "#item:event name"
+            ]
+          }
+        },
+        {
+          "name": "Boolean Attributes",
+          "description": "Boolean Attributes expression",
+          "virtual": true,
+          "doc-url": "https://lit.dev/docs/templates/expressions/#boolean-attribute-expressions",
+          "value": {
+            "kind": "expression",
+            "type": "boolean",
+            "required": true
+          },
+          "pattern": {
+            "items": {
+              "path": "/html/attributes",
+              "includeVirtual": false
+            },
+            "template": [
+              "?",
+              "$...",
+              "#item:attribute name"
+            ]
+          }
+        },
+        {
+          "name": "Properties",
+          "description": "Properties expression",
+          "virtual": true,
+          "doc-url": "https://lit.dev/docs/templates/expressions/#property-expressions",
+          "value": {
+            "kind": "expression",
+            "type": "any",
+            "required": true
+          },
+          "inject-language": "JavaScript",
+          "pattern": {
+            "items": {
+              "path": "/html/attributes",
```

Comment:
```suggestion
              "path": "/js/properties",
```
In lit, attributes can be named differently than properties. I think then path should be "/js/properties".

For example:
A `test-component`
```js
import {LitElement, html} from 'lit';
import {customElement, property} from 'lit/decorators.js';

@customElement('test-component')
export class TestComponent extends LitElement {
  @property({attribute: 'test-demo'})
  testDemo;

  render() {
    return html`${this.testDemo}`;
  }
}
```
Would be used with following usage:
```html
      <test-component test-demo="foo"></test-component>
      <test-component .testDemo="${this.bar}"></test-component>
```

Corresponding component web-types file would be something like
```json
{
  "$schema": "http://json.schemastore.org/web-types",
  "name": "...",
  "version": "...",
  "framework": "lit",
  "contributions": {
    "html": {
      "elements": [
        {
          "name": "test-component",
          "attributes": [
            {
              "name": "test-demo",
              "value": {
                "type": "string",
                "required": true
              }
            }
          ],
          "js": {
            "properties": [
              {
                "name": "testDemo",
                "value": {
                  "type": "string",
                  "required": true
                }
              }
            ]
          }
        }
      ]
    }
  }
}
```